ext/ManoptManifoldsExt/ARC_CG.jl

```diff
@@ -1,8 +1,8 @@
-function set_manopt_parameter!(M::TangentSpaceAtPoint, ::Val{:p}, v)
+function set_manopt_parameter!(M::TangentSpace, ::Val{:p}, v)
```

Comment:
I think we could even move this from extension to core Manopt.

Reply:
You are right, nice! Will do that later today (first have a lecture now soon).

Reply:
Sure. `alternating_gradient.jl` from extension should also work in core Manopt now.